amethyst_renderer/src/pipe/pipe.rs

```diff
@@ -233,11 +233,12 @@ where
 
         targets.insert("".into(), out.clone());
 
+        #[rustfmt::skip] // try is a reserved keyword in Rust 2018, must preserve keyword escape.
```

Comment:
You should probably mark this as `TODO` or `FIXME` so that it's easier to find in the future :mag:

Reply:
Yeah I was going to do that as soon as I got more info from the rustfmt project on the issue.